src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitThis = sourceString.split(';');
+  const result = {};
```

Comment:
The variable name `result` is too generic and does not describe what it holds. Consider using a more descriptive name like `stylesObject` to improve readability.